constants.py

```diff
@@ -683,6 +683,16 @@
 kOffsetDistanceRange = 1
 """meters"""
 
+kTimeOfFlightList = [
+    (2.5, 1),
+    (3.0, 1.05),
+    (3.5, 1.1),
+    (4.0, 1.15),
+    (4.5, 1.32),
+    (5.0, 1.4),
+    (5.5, 1.52),
+]
```

Comment:
Need a comment indicating units (if applicable) and explaining what these values represent.